web/src/main/webapp/js/main-template.js

```diff
@@ -158,6 +159,8 @@ $(document).ready(function (e) {
 
                 // execute query
                 initFromParams(urlParams, true);
+
+                checkInput();
```

Comment:
The checkInput(); on line 207 below now happens before the new instance here. It becomes redundant, and could probably be removed.